locales/th/php-inline.json

```diff
@@ -1,124 +1,124 @@
 {
-    "accepted": "ช่องข้อมูลนี้จะต้องรับการยอมรับ",
-    "accepted_if": "ช่องข้อมูลนี้ต้องได้รับการยอมรับเมื่อ :other มีค่าเท่ากับ :value.",
-    "active_url": "นี่ไม่ใช่ที่อยู่ URL ที่ใช้การได้",
-    "after": "นี่ต้องเป็นวันหลังจาก :date.",
-    "after_or_equal": "นี่ต้องเป็นวันหลังจากหรือเท่ากับ :date.",
-    "alpha": "ช่องข้อมูลนี้อาจจะเป็นเพียงแค่จะบรรจุจดหมาย",
-    "alpha_dash": "ช่องข้อมูลนี้อาจจะเป็นเพียงแค่จะบรรจุจดหมายของตัวเลข dashes และ underscores.",
-    "alpha_num": "ช่องข้อมูลนี้อาจจะเป็นเพียงแค่จะบรรจุจดหมายและเบอร์โทรศัพท์",
-    "array": "ช่องข้อมูลนี้จะต้องเป็นอาเรย์.",
+    "accepted": "จำเป็นต้องยอมรับฟิลด์นี้",
```

Comment:
I noticed that the "ง" character denotes the end of a sentence. The original sentence ends with a full stop, but the "ง" character is missing in the proposed version.

The original sentence contains a full stop character. Can you check it?

https://github.com/Laravel-Lang/lang/blob/main/source/framework/10.x/validation.php

Reply:
`"accepted": "จำเป็นต้องยอมรับฟิลด์นี้",` >> This one is corrected. 
There is no sentence-end punctuation in Thai. That means a period (.) is not required.